### ISSUE TITLE
can not build on mac with arm64 arch.

### ISSUE THREAD

**Issue by bsg16** [vmprof/vmprof-python#245]

**Comment by matthiasdiener** [comment]
PR #244 (which was just merged) should fix building on M1.

**Comment by nijave** [comment]
The version in pypi is old but it installs from git on Sonoma M1 fine

```
pip install git+https://github.com/vmprof/vmprof-python.git
```

**Comment by mattip** [comment]
We released 0.4.17, so installing from the source tarbll on PyPI should work now as well. Closing. It would be nice to create arm64 wheels at some point.
